recipes-browser/wpewebkit/wpewebkit.inc

```diff
@@ -101,6 +103,9 @@ PACKAGECONFIG[webxr] = "-DENABLE_WEBXR=ON,-DENABLE_WEBXR=OFF,openxr"
 
 EXTRA_OECMAKE = " -DPORT=WPE -G Ninja"
 
+# temporarily enable WPE API 1.1, for cog 0.16
+EXTRA_OECMAKE:append = " -DENABLE_WPE_1_1_API:BOOL=ON"
```

Comment:
I'd rather add it as a PACKAGECONFIG flag (e.g: `PACKAGECONFIG[wpe2api]`)  not added by default to the PACKAGECONFIG. Having this, the user can still select and test the WPE 2 API for the cog_0.17 .

BTW, that flag must be added in the wpewebkit_2.40.bb since it is only available there.

Reply:
I was thinking of a flash-fix and then flash-revert in a few days, but you prefer to have it long-term then?  I think that it makes sense.

Reply:
OK I pushed a new version.  I was working on it without having in mind what you said above, PACKAGECONFIG[wpe2api], and I did it the other way around: having like api_ver=2 as default using an option to enable api_ver=1.1.

I don't know much about the details of why the change, if there's an urgency/strong preference to push the new API for some reason or if it's better to keep 1.1 at least for one cycle for compatibility or whatever.

But since WPE was switched to use 2.0 and 1.1 is disabled by default, I think that it makes sense to do the same.  But I don't feel strongly about it at all, so fine for me if you prefer otherwise.